### PR TITLE
enable react animations for comments list

### PR DESCRIPTION
**Pull request by thiagoc7**
Just following the [guide](https://facebook.github.io/react/docs/animation.html), and using the same `element` class that already exist.

### CODE REVIEW

**Comment by rstudner** [comment]
I believe ReactCSSTransitionGroups will throw console warnings still.  It was discovered in 0.14.0, and sadly not yet fixed in 0.14.2 (it will be in 0.14.3).

Do you see warnings in your console when you do the animations?

**Comment by thiagoc7** [comment]
no warnings... checked on both 3000 and 4000 ports.
Maybe you had warnings due to using `index` as key, it just works with `unique id`.

I just don't know how to fix the issues on travis.

**Comment by rstudner** [comment]
Huh.. interesting/good.

This is the issue I was referring to:
https://github.com/facebook/react/issues/4876

File: client/server.js [hunk 1]
```diff
@@ -5,10 +5,11 @@ var WebpackDevServer = require('webpack-dev-server');
 var jade = require('jade');
 var sleep = require('sleep');
 var config = require('./webpack.client.hot.config');
+var uuid = require('node-uuid')
 
 var comments = [
-  { author: 'Pete Hunt', text: 'Hey there!' },
-  { author: 'Justin Gordon', text: 'Aloha from @railsonmaui' },
+  { author: 'Pete Hunt', text: 'Hey there!', id: uuid.v4() },
```
Comment (justin808) [inline client/server.js]: Why is this added?

File: client/app/components/CommentList.jsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ class CommentList extends React.Component {
       // http://facebook.github.io/react/docs/multiple-components.html#dynamic-children
       return (
         <Comment
-          key={index}
+          key={$$comment.get('id')}
```
Comment (justin808) [inline client/app/components/CommentList.jsx]: Why is this change necessary?

**Comment by justin808** [comment]
Hi @thiagoc7. Maybe rebase on top of master and let me know what the next step is.

File: client/server.js (hunk 1 above)
Comment (thiagoc7) [inline client/server.js]: we need a unique id for list animations

File: client/app/components/CommentList.jsx (hunk 2 above)
Comment (thiagoc7) [inline client/app/components/CommentList.jsx]: to get the unique id, or animations won't work

**Comment by justin808** [comment]
@thiagoc7 @josiasds @mamute, let's see about getting this merged.

**Comment by josiasds** [comment]
Finished on #164.
